resource_management_mobile/src/app/pages/profile/models/profile.model.ts

```diff
@@ -0,0 +1,17 @@
+export interface addUserResponse {
```

Comment:
Extend status code and message form base.model.ts

Reply:
Done